packages/datadog-plugin-jest/src/index.js

```diff
@@ -111,10 +117,13 @@ function createHandleTestEvent (tracer, testEnvironmentMetadata, instrumenter) {
       'x-datadog-sampled': 1
     })
     let testName = event.test.name
-    const context = this.getVmContext()
-    if (context) {
-      const { currentTestName } = context.expect.getState()
-      testName = currentTestName
+    let context
+    if (typeof this.getVmContext === 'function') {
```

Comment:
Since this check is done many times, it would be cleaner to move it to a new `getVmContext(this)` function so that the rest of the code can stay mostly unchanged.